lib/manageiq/appliance_console/evm_server.rb

```diff
@@ -0,0 +1,65 @@
+module ManageIQ
+  module ApplianceConsole
+    class EvmServer
+      class NotRunnableError < RuntimeError; end
+
+      class << self
+        def running?
+          service.running?
+        end
+
+        def start!(enable: false)
+          raise NotRunnableError, not_runnable_reason unless runnable?
+
+          service.start(enable)
+        end
+
+        def start(enable: false)
+          start!(:enable => enable)
+        rescue AwesomeSpawn::CommandResultError, NotRunnableError => e
+          say e.result.output
+          say e.result.error
+          say ""
```

Comment:
where is `say` defined? or is it a global method?

Reply:
I see this used everywhere, can check where it is defined though

Reply:
I assume it comes from highline globally.